global.json

```diff
@@ -1,3 +1,6 @@
 {
-    "projects": [ "src", "test", "samples" ]
+    "projects": [ "src", "test", "samples" ],
+    "sdk": {
+        "version": "1.0.0-preview1-002702"

```

Comment:
@khellang here https://blogs.msdn.microsoft.com/visualstudio/2016/05/16/announcing-updated-web-development-tools-for-asp-net-core-rc2/